package.json

```diff
@@ -55,13 +55,10 @@
   },
   "devDependencies": {
     "@silverstripe/eslint-config": "0.0.5",
-    "@silverstripe/webpack-config": "^1.0.0",
-    "babel-core": "^6.26.3",
+    "@silverstripe/webpack-config": "^1.3.0",
     "babel-jest": "^23.6.0",
-    "babel-loader": "^7.0.0",
     "babel-polyfill": "^6.26.0",
```

Comment:
If you're removing these dependencies because they're unused, there might be more you can remove:

![image](https://user-images.githubusercontent.com/541886/68359223-15b75700-0180-11ea-9152-50b1f3059737.png)

Reply:
When in doubt, I'd rather keep them there otherwise, I'll have to spend time to make sure removing them doesn't actually break anything.

Because of the way our projects are set up it's not always clear what we can ditch.